src/Utilities/CssConcatenator.php

```diff
@@ -42,16 +42,9 @@
 class CssConcatenator
 {
     /**
-     * Array of media rules in order.  Each element is an object with the following properties:
-     * - string `media` - The media query string, e.g. "@media screen and (max-width:639px)", or an empty string for
-     *   rules not within a media query block;
-     * - \stdClass[] `ruleBlocks` - Array of rule blocks in order, where each element is an object with the following
-     *   properties:
-     *   - mixed[] `selectorsAsKeys` - Array whose keys are selectors for the rule block (values are of no
-     *     significance);
-     *   - string `declarationsBlock` - The property declarations, e.g. "margin-top: 0.5em; padding: 0".
+     * Array of media rules in order.
      *
-     * @var \stdClass[]
+     * @var CssConcatenator\MediaRule[]
```

Comment:
Please let's `use` the new classes in `CssConcatenator` so the prefix is not needed.

Reply:
It's good to have a list of `use`s at the top of the file to clearly identify what the dependencies are :)